assets/js/app.js

```diff
@@ -22,6 +30,9 @@ import "./filter_trade_form";
 import "./filter_players_list";
 import "./confirm_submit";
 
+let liveSocket = new LiveSocket("/live");
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)